src/ocean/text/convert/Formatter.d

```diff
@@ -532,8 +536,10 @@ private void handle (T) (T v, FormatInfo f, FormatterSink sf, ElemSink se)
     }
 
     // Arrays (dynamic and static)
-    else static if (is (T A : A[]))
+    else static if (isBasicArrayType!(T))
```

Comment:
Does
```d
 static if (is(ElementTypeOf!(T) A))
```
work? ;)

Reply:
Yep (great idea btw!), but it is not very suitable here as `ElementTypeOf` also works on associative arrays.